website/source/docs/providers/kubernetes/r/secret.html.markdown

```diff
@@ -0,0 +1,69 @@
+---
+layout: "kubernetes"
+page_title: "Kubernetes: kubernetes_secret"
+sidebar_current: "docs-kubernetes-resource-secret"
+description: |-
+  The resource provides mechanisms to inject containers with sensitive information while keeping containers agnostic of Kubernetes.
+---
+
+# kubernetes_secret
+
+The resource provides mechanisms to inject containers with sensitive information, such as passwords, while keeping containers agnostic of Kubernetes.
+Secrets can be used to store sensitive information either as individual properties or coarse-grained entries like entire files or JSON blobs.
+The resource will by default create a secret which is available to any pod in the specified (or default) namespace.
+Read more about security properties and risks involved with using Kubernetes secrets: https://kubernetes.io/docs/user-guide/secrets/#security-properties
+
+**Note:** All arguments including the secret data will be stored in the raw state as plain-text.
+[Read more about sensitive data in state](/docs/state/sensitive-data.html).
```

Comment:
To make the two paragraphs (previous one starting with `Read more about security properties ...`) stand out, would you mind prefixing each with `~> `?

That makes the paragraph turn into this yellow rectangle:

![screen shot 2017-03-30 at 09 02 27](https://cloud.githubusercontent.com/assets/287584/24493709/b7420356-1527-11e7-95a4-49664d1592ea.png)